### DISCUSSION TITLE
information looks too scientific

### DISCUSSION THREAD

**Discussion by Pinkyponki** [lsc-hubs/kenya-hub#15] [General]
Considering the end user/audience, the information looks too scientific and may not be easily utilized/understood by the Extension/farmer in the required manner. Kindly simplify the content as much as possible

**Comment by pvgenuchten** [comment]
hi @Pinkyponki, good to hear your thoughts on this. Personally I think we should not limit our content in order to extend our reach, in stead we should guide specific users to relevant content as soon as possible. 

What content would be relevant to the user group you intend to service? How would that user group prefers to find that content between the other content (are you a farmer? click here)
